sdk-api-src/content/winioctl/ns-winioctl-usn_record_v2.md

```diff
@@ -85,7 +85,7 @@ The size in bytes of any change
 <td>
 <pre>  MaximumChangeJournalRecordSize = 
       ( MaximumComponentLength * sizeof(WCHAR) 
-        + sizeof(USN_RECORD)   - sizeof(WCHAR) );
+        + sizeof(USN_RECORD)   + sizeof(WCHAR) );
```

Comment:
I don't think the subtraction here was a typo. The `FileName` field in `USN_RECORD` is a 1-element array, so one `WCHAR` is already included in `sizeof(USN_RECORD)`. The subtraction compensates for that. In other words, the result is the same as `FIELD_OFFSET(USN_RECORD, FileName)`.